revisions/N0/src/text/dialog_3.asm

```diff
@@ -128,586 +128,1074 @@ Dialog115::
+    db "hvorfor, fordi  "
+    db "jeg bare er et  "
+    db "barn!@"
+;    db "Well, it seems  "
+;    db "that after you  "
+;    db "save, you will  "
+;    db "start at the    "
+;    db "last door you   "
+;    db "went through... "
+;    db "I'm not really  "
+;    db "sure why that is"
+;    db "'cause I'm just "
+;    db "a kid!@"
 
 Dialog11A::
-    db "I heard that you"
-    db "can press SELECT"
-    db "to look at the  "
-    db "island map...   "
-    db "But, I don't    "
-    db "understand what "
-    db "they mean by    "
-    db "that...@"
+    db "Jeg har hørt at "
+    db "du kan trykke på"
+    db "SELECT for å se "
+    db "på kartet over  "
+    db "øya_… Men jeg   "
+    db "skjønner ikke   "
+    db "hva det betyr_…@"
+;    db "I heard that you"
+;    db "can press SELECT"
+;    db "to look at the  "
+;    db "island map...   "
+;    db "But, I don't    "
+;    db "understand what "
+;    db "they mean by    "
+;    db "that...@"
 
 Dialog11B::
-    db "When you're     "
-    db "running out of  "
-    db "Hearts, you'd   "
-    db "better enter a  "
-    db "house or cave..."
-    db "Why?  I have no "
-    db "idea, I'm just a"
-    db "kid!@"
+    db "Hvis du begynner"
+    db "å gå tom for    "
+    db "hjerter er det  "
+    db "lurt å gå inn i "
+    db "et hus eller en "
+    db "hule_… Hvorfor? "
+    db "Aner ikke, jeg  "
+    db "er bare et barn!@"
+;    db "When you're     "
+;    db "running out of  "
+;    db "Hearts, you'd   "
+;    db "better enter a  "
+;    db "house or cave..."
+;    db "Why?  I have no "
+;    db "idea, I'm just a"
+;    db "kid!@"
 
 Dialog11C::
-    db "Hey, dude! What "
-    db "do you think of "
-    db "Marin?  Uhh...  "
-    db "I don't know,   "
-    db "I'm just a kid!@"
+    db "Halla! Hva synes"
+    db "du om Marin?    "
+    db "Ehh_… Jeg vet   "
+    db "ikke, jeg er    "
+    db "bare et barn!@"
+;    db "Hey, dude! What "
+;    db "do you think of "
+;    db "Marin?  Uhh...  "
+;    db "I don't know,   "
+;    db "I'm just a kid!@"
 
 Dialog11D::
-    db "Where are you   "
-    db "from, brother?  "
-    db "...Outside the  "
-    db "island?  What is"
-    db "'outside?'  I've"
-    db "never thought   "
-    db "about it...@"
+    db "Hvor er du fra, "
+    db "bror? _… Fra    "
+    db "utenfor øya? Hva"
+    db "er «utenfor»?   "
+    db "Det har jeg     "
+    db "aldri tenkt på  "
+    db "før_…@"
+;    db "Where are you   "
+;    db "from, brother?  "
+;    db "...Outside the  "
+;    db "island?  What is"
+;    db "'outside?'  I've"
+;    db "never thought   "
+;    db "about it...@"
 
 Dialog11E::
-    db "The giant egg on"
-    db "top of Tamaranch"
-    db "Mountain?  They "
-    db "say the Wind    "
-    db "Fish is sleeping"
-    db "inside of it... "
-    db "Why?  I don't   "
-    db "know either...@"
+    db "Det svære egget "
+    db "på toppen av    "
+    db "Tindeggen? Det  "
+    db "sies at Vind-   "
+    db "fisken sover in-"
```

Comment:
Hva med å skrive "sover på innsiden" for å slippe den litt stygge orddelingen av in-ne?

Og kanskje veit i stedet for vet? Om vi først skal skrive Neigu. Neigu om jeg veit...

Reply:
```suggestion
    db "fisken sover på "
```